### PR TITLE
docs: add missing import for `@react-aria/switch`

### PR DESCRIPTION
**Pull request by Dev-CasperTheGhost**
Closes 

This PR simply adds a missing import for `@react-aria/switch`'s documentation.

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [x] Updated documentation (if it already exists for this component).
- [x] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

**Comment by Dev-CasperTheGhost** [comment]
CLA is now signed, not sure if the Github action needs to be run again..

**Comment by snowystinger** [comment]
It has a reasonable error
> @parcel/transformer-babel: Identifier 'useSwitch' has already been declared (7:8)
```
/home/circleci/react-spectrum/packages/@react-aria/switch/docs/useSwitch.mdx:7:8
  6 | import {useSwitch} from '@react-aria/switch';
> 7 | import {useSwitch} from '@react-aria/switch';
>   |        ^
  8 | import {VisuallyHidden} from '@react-aria/visually-hidden';
  9 | import {useToggleState} from '@react-stately/toggle';
```

File: packages/@react-aria/switch/docs/useSwitch.mdx [hunk 1]
```diff
@@ -91,6 +91,7 @@ indicate focus. The focus ring is only visible when the user is interacting with
 not with a mouse or touch.
 
 ```tsx example
+import {useSwitch} from '@react-aria/switch';
```
Comment (snowystinger) [inline packages/@react-aria/switch/docs/useSwitch.mdx:94]: this is already included here https://github.com/adobe/react-spectrum/blame/main/packages/%40react-aria/switch/docs/useSwitch.mdx#L22
and it gets surfaced at the top of the docs
Screen Shot 2021-12-07 at 4 19 09 PM

you'll find this across every doc page

Comment (snowystinger) [reply packages/@react-aria/switch/docs/useSwitch.mdx:94]: you might be able, in this instance, remove it from the line i pointed to, as long as it's in an example, it'll be imported. our docs can only import each one once

Comment (Dev-CasperTheGhost) [reply packages/@react-aria/switch/docs/useSwitch.mdx:90]: oh yeah, makes sense. I've now removed that top import so it's only in the example

**Review by snowystinger — APPROVED** [review]
thank you!

**Review by LFDanLu — APPROVED** [review]
LGTM, looks fine locally.

**Comment by dannify** [comment]
Thanks for the contribution @Dev-CasperTheGhost ⭐

**Pull request merged**